app/src/test/java/org/simple/clinic/setup/SetupActivityUpdateTest.kt

```diff
@@ -24,7 +24,7 @@ class SetupActivityUpdateTest {
 
     updateSpec
         .given(defaultModel)
-        .whenEvent(UserDetailsFetched(hasUserCompletedOnboarding = true, loggedInUser = None))
+        .whenEvent(UserDetailsFetched(hasUserCompletedOnboarding = true, loggedInUser = None, userSelectedCountry = None))
```

Comment:
Same as above.

Reply:
Added a bunch of factory functions in the test to better expose intent.